source/gui/settingsDialogs.py

```diff
@@ -2710,6 +2711,28 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 
 		self._onSoundVolChange(None)
 
+		# Translators: This is a label for the sound split combo box in the Audio Settings dialog.
+		soundSplitLabelText = _("Sound split mode:")
+		self.soundSplitComboBox = sHelper.addLabeledControl(
+			soundSplitLabelText,
+			wx.Choice,
+			choices=[mode.displayString for mode in audio.SoundSplitState]
+		)
+		self.bindHelpEvent("SelectSoundSplitMode", self.soundSplitComboBox)
+		index = config.conf["audio"]["soundSplitState"]
+		self.soundSplitComboBox.SetSelection(index)
+
+		# Translators: This is a label for the sound split Toggle Mode combo box in the Audio Settings dialog.
+		soundSplitToggleModeLabelText = _("NVDA+Alt+S command behavior:")

```

Comment:
Also:

3. Question: With this PR, we would now be able to select a subset of values for a second cycle command, the first being speech mode. If NV Access accept this and if it becomes more frequent, would it be worth to dedicate a specific setting panel for such customization? E.g. command customization.

Reply:
> 1. Please, do not use NVDA+Alt+S, but rather the name of the command (e.g. as seen in input gesture dialog); this way, the name of this control always remains valid and it is also more explicit.
Done
> 2. More generally, would it be interesting to convert this control to a checkable list box as done for the available speech mode selection? That may be done now or later in case [Sound split #16071 (comment)](https://github.com/nvaccess/nvda/pull/16071#issuecomment-1902237001) is accepted.
I like this idea - just implemented that. Also this allows to implement suggestion by @cary-rowen.

Reply:
I converted combobox to list of checkbox as you suggested. As for which panel to put it on, it seems to me that sound split modes and speech modes customizations should live on different panels because they are related to semantically different commands.